.gitignore

```diff
@@ -6,6 +6,7 @@
 data
 runs
 examples/saved_models
+vulcanai2/tests/models/saved_models/
```

Comment:
since you will delete the file after saving, you won't need this anymore

Reply:
👍